OptimizelySDK.Tests/OptimizelyConfigTests/OptimizelyConfigTest.cs

```diff
@@ -177,6 +176,33 @@ public void TestGetOptimizelyConfigSDKAndEnvironmentKeyDefault()
             Assert.AreEqual(optimizelyConfig.EnvironmentKey, "");
         }
 
+        [Test]
+        public void TestGetOptimizelyConfigWithOdpIntegration()
```

Comment:
Can we move all these tests to ProjectConfigTest? They're not related to OptimizelyConfig.